docs/_news/news12.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: news
+title: New for Codewind 0.12.0
+description: New for Codewind 0.12.0
+keywords: news, new, updates, update, version, IntelliJ
+duration: 1 minute
+permalink: news12
+---
+
+## Codewind 0.12.0
+Thursday 14, May 2020
+
+#### ✨ New Features and Highlights for 0.12.0 ✨
+**IntelliJ Tech Preview Updates**
+We're working to bring you a true Codewind experience with IntelliJ and are pleased to present:
+
+- Codewind for IntelliJ [supports the application metrics dashboard and the **Inject metrics** function](https://github.com/eclipse/codewind/issues/2386).
+- You can [open a shell into the running application container](https://github.com/eclipse/codewind/issues/2030)
+- You will be [notified of log changes, even when the log is not currently selected](https://github.com/eclipse/codewind/issues/2269).
+- When a project is opened in a new window, via actions such as reload due to new projoect creation or restart in debug mode (etc), [the project will be pre-selected in the Codewind view](https://github.com/eclipse/codewind/issues/2037).
+
+**Codewind Quick Guides**
+
+Introducing... Quick Guides! Now that we have [this framework in place](https://github.com/eclipse/codewind-docs/pull/553), we hope to make it easier for you to get started. Check out our first two available [guides](https://www.eclipse.org/codewind/guides.html) today!
```

Comment:
- I understand the `...` to indicate a pause, but I'd remove the ellipsis.
- Lower case "quick guides" in the body text.